packages/js/test/plugins/auctionHouseModule/createBid.test.ts

```diff
@@ -208,3 +210,158 @@ test('[auctionHouseModule] create public receipt-less bid but cannot fetch it af
+    })
+    .run();
+
+  // Then we created and returned the new Bid with appropriate defaults.
+  t.equal(bid.tradeStateAddress, buyerTradeState);
+  t.same(bid.price, sol(1));
+  t.same(bid.tokens, token(1));
+  t.ok(bid.isPublic);
+});
+
+test('[auctionHouseModule] it throws an error if Buy is not included in Auctioneer scopes', async (t: Test) => {
+  // Given we have an NFT.
+  const mx = await metaplex();
+  const seller = await createWallet(mx);
+  const nft = await createNft(mx, {}, { owner: seller.publicKey });
+
+  const auctioneerAuthority = Keypair.generate();
+
+  // Create Auctioneer Auction House to only allow Sell.
+  const { client } = await createAuctionHouse(mx, auctioneerAuthority, {
+    auctioneerScopes: [AuthorityScope.Sell],
+  });
+
+  // When we create a private bid on that NFT for 1 SOL.
+  const promise = client
+    .bid({
+      mintAccount: nft.mintAddress,
+      seller: seller.publicKey,
+      price: sol(1),
+    })
+    .run();
+
+  // Then we expect an error.
+  await assertThrows(
+    t,
+    promise,
+    /The Auctioneer does not have the correct scope for this action/
+  );
+});
+
+test('[auctionHouseModule] it allows to Buy after Auctioneer scope update', async (t: Test) => {
+  // Given we have an NFT.
+  const mx = await metaplex();
+  const seller = await createWallet(mx);
+  const nft = await createNft(mx, {}, { owner: seller.publicKey });
+
+  const auctioneerAuthority = Keypair.generate();
+
+  // Create Auctioneer Auction House to only allow Sell.
+  const { auctionHouse, client } = await createAuctionHouse(
+    mx,
+    auctioneerAuthority,
+    {
+      auctioneerScopes: [AuthorityScope.Sell],
+    }
+  );
+
+  // When we update scope to allow Buy.
+  await mx
+    .auctions()
+    .updateAuctionHouse(auctionHouse, {
+      auctioneerAuthority: auctioneerAuthority.publicKey,
+      auctioneerScopes: [AuthorityScope.Sell, AuthorityScope.Buy],
+    })
+    .run();
+
+  // When we create a private bid on that NFT for 1 SOL.
+  const { bid, buyerTradeState } = await client
+    .bid({
+      mintAccount: nft.mintAddress,
+      seller: seller.publicKey,
+      price: sol(1),
+    })
+    .run();
+
+  // Then we still get a listing model.
+  t.equal(bid.tradeStateAddress, buyerTradeState);
+});
+
+test('[auctionHouseModule] it throws an error if Auctioneer Authority is not provided in Auctioneer Bid', async (t: Test) => {
+  // Given we have an NFT.
+  const mx = await metaplex();
+  const seller = await createWallet(mx);
+  const nft = await createNft(mx);
+
+  const auctioneerAuthority = Keypair.generate();
+
+  const { auctionHouse } = await mx
+    .auctions()
+    .createAuctionHouse({
+      sellerFeeBasisPoints: 200,
+      auctioneerAuthority: auctioneerAuthority.publicKey,
+    })
+    .run();
+  // Create a client for Auction House, but don't provide auctioneerAuthority.
+  const client = mx.auctions().for(auctionHouse);
+
+  // When we create a private bid on that NFT for 1 SOL.
```

Comment:
Same here, just improving the comments.

```suggestion
  // And an Auctioneer Auction House.
  const auctioneerAuthority = Keypair.generate();
  const { auctionHouse } = await mx
    .auctions()
    .createAuctionHouse({
      sellerFeeBasisPoints: 200,
      auctioneerAuthority: auctioneerAuthority.publicKey,
    })
    .run();
 
  // When we create a client for that Auction House without providing the auctioneerAuthority.
  const client = mx.auctions().for(auctionHouse);

  // And we create a private bid on that NFT for 1 SOL.
```